Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift

```diff
@@ -78,11 +83,13 @@ class RubricCircleView: UIView {
             addSubview(button)
             buttons.append(button)
 
+            button.isUserInteractionEnabled = false
```

Comment:
I decided to leave the buttons in and just disable them for now so VO would inform the user the button is disabled.